### ISSUE TITLE
[5.2] Filter bug with custom admin menu item

### ISSUE THREAD

**Issue by brianteeman** [joomla/joomla-cms#44407]
### Steps to reproduce the issue
Create an admin menu module
Create a link of type menu items
Publish this additional admin menu module in position menu
Click on the newly created menu item link

### Expected result
The site/admin filter works

### Actual result
The filter never changes from site

**Comment by hytsch** [comment]
Too bad this bug was not fixed with 5.2.1. As long as the bug persists, we have to stick with 5.1 ;(

**Comment by alikon** [comment]
should this be solved with #44426

**Comment by brianteeman** [comment]
No. #44426 is to fix the code Hannes removed in error from com_content. Possibly this issue can be fixed with another PR to fix the code removed from com_modules

**Comment by brianteeman** [comment]
Found the problem - it doesnt switch because the second filter to select the menu needs to be unselected

**Comment by brianteeman** [comment]
Perhaps someone who understand the javascript can fix this. 
Basically what is needed is for the second filter to be set to "- Select Menu -" whenever the first filter is changed from Site to Admin and vice versa

![Image](https://github.com/user-attachments/assets/093cca35-7f72-451e-8c64-86c837ac56cb)

**Comment by Hemang360** [comment]
@brianteeman I was able to reproduce the issue without adding a new menu.
Navigating to Menu → Bottom Menu does not update the site/admin filter, which remains stuck on "site."

**Comment by brianteeman** [comment]
@Hemang360 thats because you have installed the sample data which includes multiple menus. My instructions were for everyone so they may not have multiple menus already.
